.ci/update-github-page.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+set -e
+
+source ./.ci/util.sh
+
+checkForVariable "GITHUB_TOKEN"
+
+checkout_from https://github.com/checkstyle/contribution
+
+cd .ci-temp/contribution/releasenotes-builder
+mvn -e --no-transfer-progress clean compile package
+cd ../../../
+
+if [ -d .ci-temp/checkstyle ]; then
+  cd .ci-temp/checkstyle/
+  git reset --hard origin/master
+  git pull origin master
+  git fetch --tags
+  cd ../../
+else
+  cd .ci-temp/
+  git clone https://github.com/checkstyle/checkstyle
+  cd ../
+fi
+
+cd .ci-temp/checkstyle
+
+curl \
+ https://api.github.com/repos/checkstyle/checkstyle/releases \
+ -H "Authorization: token $GITHUB_TOKEN" \
+ -o /var/tmp/cs-releases.json
+
+TARGET_RELEASE_NUM=10.1
+TARGET_RELEASE_INDEX=$(cat /var/tmp/cs-releases.json | \
+    jq "[.[].tag_name] | to_entries | .[] | \
+    select(.value==\"checkstyle-$TARGET_RELEASE_NUM\") | .key")
+echo TARGET_RELEASE_INDEX="$TARGET_RELEASE_INDEX"
+
+PREVIOUS_RELEASE_INDEX=$(($TARGET_RELEASE_INDEX+1))
+echo PREVIOUS_RELEASE_INDEX="$PREVIOUS_RELEASE_INDEX"
+
+END_REF=$(cat /var/tmp/cs-releases.json | jq -r ".[$TARGET_RELEASE_INDEX].tag_name")
+START_REF=$(cat /var/tmp/cs-releases.json | jq -r ".[$PREVIOUS_RELEASE_INDEX].tag_name")
+
+echo START_REF="$START_REF"
+echo END_REF="$END_REF"
+
+cd ../
+
+BUILDER_RESOURCE_DIR="contribution/releasenotes-builder/src/main/resources/com/github/checkstyle"
+
+java -jar contribution/releasenotes-builder/target/releasenotes-builder-1.0-all.jar \
+     -localRepoPath checkstyle \
+     -remoteRepoPath checkstyle/checkstyle \
+     -startRef "$START_REF" \
+     -endRef "$END_REF" \
+     -releaseNumber "$CS_RELEASE_VERSION" \
+     -githubAuthToken "$READ_ONLY_TOKEN" \
+     -generateGitHub \
+     -gitHubTemplate $BUILDER_RESOURCE_DIR/templates/github_post.template
+
+echo ==============================================
+echo "GITHUB PAGE:"
+echo ==============================================
+CONTENT=$(cat github_post.txt)
+echo "$CONTENT"
+
+UPDATED_CONTENT=$(awk '{printf "%s\\n", $0}' github_post.txt |  sed "s/\`/'/g")
+echo "$UPDATED_CONTENT"
+
+RELEASE_ID=$(curl -s https://api.github.com/repos/checkstyle/checkstyle/releases/latest \
+                       | jq ".id")
+echo RELEASE_ID="$RELEASE_ID"
```

Comment:
not a latest release, but a version from script parameter.

Reply:
But how will we fetch the RELEASE_ID then? For latest, we can easily get using: jq (73283356 - latest)

Reply:
But we want to edit the Github page for the latest release so we will need the RELEASE_ID of the latest release.

Reply:
if you run `.ci/update-github-page.sh 10.1`
and in code you have 
`TARGET_RELEASE_NUM=$1`

script is not working with latest release.
it is capable to update any release.

Reply:
Changes Done